FreewheelSampleApp/app/src/main/java/com/ooyala/sample/players/CustomConfiguredFreewheelPlayerActivity.java

```diff
@@ -102,6 +104,19 @@ public void update(Observable arg0, Object arg1) {
     if (arg1 == OoyalaPlayer.TIME_CHANGED_NOTIFICATION) {
       return;
     }
+
+
+
+    // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
+    String text="Notification Received: " + arg1 + " - state: " + player.getState() + "count: " +count;
+
+    // Write the event text along with event count to log file in sdcard if the log file exists
+    SDCardLogcatOoyalaEventsLogger Playbacklog= new SDCardLogcatOoyalaEventsLogger();
```

Comment:
I do generally like the idea of simply making a new() one every time as a general way to write code. Of course, the flip side is then I wonder if there's a lot of overhead in the constructor. :-) When I go look at http://github.services.ooyala.net/playback-app/android-sdk/pull/295 it looks to me like that in fact it could all just be static methods, and then there's no need for a constructor at all?